src/shared/Progression.h

```diff
@@ -27,8 +27,10 @@
 #define CLIENT_BUILD_1_8_4 4878
 #define CLIENT_BUILD_1_9_4 5086
 #define CLIENT_BUILD_1_10_2 5302
+#define CLIENT_BUILD_1_11_0 5428
```

Comment:
Why are you adding 1.11.0 ? We do not support this build. Only the last version of each major patch, not the minor patches.

Reply:
All non supported build will be catch on patch downloader. If patch isn't available, client will show invalid client version error. I added this build to not user a "magic number" in code.